### PR TITLE
[Mac] Make boxes not take focus ever

### PR DESCRIPTION
**Pull request by rodrmoya**
If not, they participate in the key view loop, which makes focus being lost
on TAB navigation when there are boxes in the window.

### CODE REVIEW

**Comment by rodrmoya** [comment]
Setting it in Initialize, throws this:

```
ERROR [2019-01-16 16:52:02Z]: An unhandled exception has occurred. Terminating Visual Studio? False
System.NullReferenceException: Object reference not set to an instance of an object
  at Xwt.Mac.ViewBackend.get_Widget () [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt.XamMac/Xwt.Mac/ViewBackend.cs:116 
  at Xwt.Mac.ViewBackend.set_CanGetFocus (System.Boolean value) [0x00008] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt.XamMac/Xwt.Mac/ViewBackend.cs:179 
  at Xwt.Mac.BoxBackend.Initialize () [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt.XamMac/Xwt.Mac/BoxBackend.cs:42 
  at Xwt.Mac.ViewBackend.Xwt.Backends.IWidgetBackend.Initialize (Xwt.Backends.IWidgetEventSink sink) [0x00008] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt.XamMac/Xwt.Mac/ViewBackend.cs:73 
  at Xwt.Widget+WidgetBackendHost.OnBackendCreated () [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt/Widget.cs:119 
  at Xwt.Backends.BackendHost.LoadBackend () [0x00099] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt.Backends/BackendHost.cs:170 
  at Xwt.Backends.BackendHost.get_Backend () [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt.Backends/BackendHost.cs:93 
  at Xwt.Backends.BackendHost`2[T,B].get_Backend () [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt.Backends/BackendHost.cs:55 
  at Xwt.Widget.get_Backend () [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt/Widget.cs:376 
  at Xwt.Widget.OnPreferredSizeChanged () [0x00008] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt/Widget.cs:1554 
  at Xwt.Box.set_Spacing (System.Double value) [0x0001f] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt/Box.cs:86 
  at (wrapper remoting-invoke-with-check) Xwt.Box.set_Spacing(double)
  at Xwt.Mac.DialogBackend.InitializeBackend (System.Object frontend, Xwt.Backends.ApplicationContext context) [0x0000a] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt.XamMac/Xwt.Mac/DialogBackend.cs:66 
  at MonoDevelop.MacIntegration.ThemedMacDialogBackend.InitializeBackend (System.Object frontend, Xwt.Backends.ApplicationContext context) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/addins/MacPlatform/MacPlatform.cs:1342 
  at Xwt.Backends.BackendHost.LoadBackend () [0x0007c] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt.Backends/BackendHost.cs:169 
  at Xwt.Backends.BackendHost.get_Backend () [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt.Backends/BackendHost.cs:93 
  at Xwt.Backends.BackendHost`2[T,B].get_Backend () [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt.Backends/BackendHost.cs:55 
  at Xwt.Window.get_Backend () [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt/Window.cs:61 
  at Xwt.Window.UpdatePadding () [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt/Window.cs:116 
  at Xwt.Window.set_Padding (Xwt.WidgetSpacing value) [0x00008] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt/Window.cs:78 
  at Xwt.Window..ctor (Xwt.WidgetSpacing initialPadding) [0x0001d] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt/Window.cs:57 
  at Xwt.Window..ctor () [0x00000] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt/Window.cs:50 
  at Xwt.Dialog..ctor () [0x00000] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt/Dialog.cs:40 
  at MonoDevelop.Ide.Projects.NewSolutionRunConfigurationDialog..ctor () [0x00000] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Projects/NewSolutionRunConfigurationDialog.cs:37 
  at (wrapper remoting-invoke-with-check) MonoDevelop.Ide.Projects.NewSolutionRunConfigurationDialog..ctor()
  at MonoDevelop.Ide.Commands.SetStartupProjectsHandler+<>c__DisplayClass1_1.<Run>b__0 () [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Commands/ProjectCommands.cs:149 
  at Xwt.Toolkit.Invoke (System.Action a) [0x00010] in /Users/rodrigo/Projects/xamarin/monodevelop/main/external/xwt/Xwt/Xwt/Toolkit.cs:470 
```

**Comment by sevoku** [comment]
Hhm, this is strange, did you try setting it before or after the `WidgetView` instantiation in https://github.com/mono/xwt/blob/7270918736d5392089bbcad9022187da54b47b11/Xwt.XamMac/Xwt.Mac/BoxBackend.cs#L42?

**Comment by rodrmoya** [comment]
both, but let me try again

**Comment by rodrmoya** [comment]
ugh, setting it *after* indeed works. I guess I didn't recompile correctly last time, so fixing PR
